Sources/TurboNavigationDelegate.swift

```diff
@@ -28,31 +34,14 @@ public protocol TurboNavigationDelegate: AnyObject {
     func sessionDidLoadWebView(_ session: Session)
 }
 
-public extension TurboNavigationDelegate {
```

Comment:
I've never liked protocol default implementations because they're too invisible sometimes. Since we'd like to nudge people towards using TurboNavigatingCoordinator, I've moved the default behavior there. That way users get the default behavior by creating a `TurboNavigatingCoordinator` and we increase visibility of defaults there.